pages/kit-docs/commands.mdx

```diff
@@ -314,3 +317,7 @@ drizzle-kit studio
 
 drizzle-kit studio --port 3000 --verbose
 ```
+
+<Button as="a">
+  <a href="http://0.0.0.0:3000">🖥 Open Drizzle Studio</a>
```

Comment:
it should either open https://demo.drizzle.studio or we now have https://local.drizzle.studio

we should potentially have 2 and first button should be `open demo drizzle studio`